strato

```diff
@@ -117,6 +123,15 @@ then
     exit 2
 fi
 
+STRATOGS_REQUIRED_VERSION=$(grep strato-getting-started-min-version docker-compose.yml | awk -F":" '{print $NF}')
+if [ ${STRATOGS_REQUIRED_VERSION} ]; then
+  if awk -v VER=$(<VERSION) -v REQ_VER=${STRATOGS_REQUIRED_VERSION} 'BEGIN { if (VER < REQ_VER) { exit 0 } else { exit 1 }}'
```

Comment:
This can be simplified to lose the `if`:

```
~/w/strato-getting-started ❯❯❯ awk -v VER=3.6 -v REQ_VER=3.5 'BEGIN {exit (VER < REQ_VER)}' 
~/w/strato-getting-started ❯❯❯ echo $?
0
~/w/strato-getting-started ❯❯❯ awk -v VER=3.6 -v REQ_VER=3.7 'BEGIN {exit (VER < REQ_VER)}'
~/w/strato-getting-started ❯❯❯ echo $?
1
```